pkg/arvo/sys/hoon.hoon

```diff
@@ -9865,6 +9865,8 @@
                   next
```

Comment:
@joemfb What are "bridges" (q.zot, processed in next)? Are they emitted by anything? In general, what part of the parser produces tunes?

Reply:
Bridges are "imports", resulting from `=,`. The `%tune` AST node is produced in the macro-expansion of `%tstr`, and a tune is constructed (with `+busk`) when inferring the type of a `%tscm`.